osu.Game/Screens/Play/HUDOverlay.cs

```diff
@@ -259,7 +259,9 @@ protected virtual ScoreCounter CreateScoreCounter() => new ScoreCounter(6)
             Margin = new MarginPadding { Top = 20, Right = 10 },
         };
 
-        protected virtual HitErrorDisplay CreateHitErrorDisplayOverlay() => new HitErrorDisplay(scoreProcessor, drawableRuleset.Objects.FirstOrDefault()?.HitWindows);
+        protected virtual HitErrorDisplay CreateHitErrorDisplayOverlay() => new HitErrorDisplay(
+            scoreProcessor,
+            drawableRuleset.Objects.Concat(drawableRuleset.Objects.SelectMany(h => h.NestedHitObjects)).FirstOrDefault(h => h.HitWindows != null)?.HitWindows);
```

Comment:
was there a specific scenario for requiring the nested logic here?

Reply:
Yes - sliders. They don't have hitwindows anymore but their heads/tails do.